mailbox/backup/src/main/java/org/apache/james/mailbox/backup/Archive.java

```diff
@@ -23,15 +23,14 @@
 import java.util.List;
 import java.util.stream.Stream;
 
-import org.apache.james.mailbox.store.mail.model.Mailbox;
-import org.apache.james.mailbox.store.mail.model.MailboxMessage;
+import org.apache.james.mailbox.model.MessageResult;
 
-public interface Backup {
+public interface Archive {
```

Comment:
`Archive` is confusing because one can think it's a piece of data. Would you mind renaming it into ArchiveService?

Reply:
It makes sense, i will rename it.